l10n_br_sale/models/res_config.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+# Copyright (C) 2014  Renato Lima - Akretion
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+
+from openerp import SUPERUSER_ID
+from openerp.osv import orm, fields
+
+
+class SaleConfiguration(orm.TransientModel):
+    _inherit = 'sale.config.settings'
+    _columns = {
+        'copy_note': fields.boolean(
+            u'Copiar Observações nos Documentos Fiscais'),
+    }
```

Comment:
would be cool to have that one in the v8 api. But as we are in a rush to get the OCA branch merged we can leave that to a next PR.